src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
+        "name": "Documentation",
+        "type": "submenu",
+        "ref": "documentation",
+        "children": [
+            {
+                "name": "Getting started",
+                "type": "listfiles",
+                "ref": "docgettingstarted",
+                "children": [
+                    {
+                        "name": "Why Elektra?",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/WHY.md"
+                        },
+                        "slug": "why-elektra"
+                    },
+                    {
+                        "name": "Big Picture",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/BIGPICTURE.md"
+                        },
+                        "slug": "big-picture"
+                    },
+                    {
+                        "name": "Compiling",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/COMPILE.md"
+                        },
+                        "slug": "compiling"
+                    },
+                    {
+                        "name": "Installation",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/INSTALL.md"
+                        },
+                        "slug": "installation"
+                    },
+                    {
+                        "name": "Tutorials",
+                        "type": "ref",
+                        "options": {
+                            "path": "tutorials"
+                        }
+                    }
+                ]
+            },
+            {
+                "name": "Tutorials",
+                "type": "listfiles",
+                "ref": "tutorials",
+                "children": [
+                    {
+                        "name": "README",
+                        "type": "file",
+                        "options": {
+                            "path": "doc/tutorials/README.md"
```

Comment:
Why not use the automatic listing of tutorials via the README? (can be changed later, though)